lib/circular_percent_indicator.dart

```diff
@@ -408,9 +416,16 @@ class _CirclePainter extends CustomPainter {
     }
 
     _paintLine.color = progressColor;
-    _paintLine.style = PaintingStyle.stroke;
-    _paintLine.strokeWidth = lineWidth;
+    _paintLine.style = PaintingStyle.fill;
+    _paintLine.strokeWidth = lineWidth - 2;
```

Comment:
why -2 here?

Reply:
Just for border size ... I think if it is dynamic, another problems will appear